library/src/main/java/com/getstream/sdk/chat/adapter/AttachmentListItemAdapter.java

```diff
@@ -22,27 +22,29 @@
     private MessageViewHolderFactory factory;
     private Context context;
     private MessageListItem.MessageItem messageListItem;
-    private List<Attachment> attachments;
+    private List<AttachmentListItem> attachments;
     private MessageListViewStyle style;
     private MessageListView.AttachmentClickListener attachmentClickListener;
     private MessageListView.MessageLongClickListener longClickListener;
     private MessageListView.GiphySendListener giphySendListener;
     private MessageListView.BubbleHelper bubbleHelper;
 
 
-    public AttachmentListItemAdapter(Context context, @NonNull MessageListItem.MessageItem messageListItem, @NonNull MessageViewHolderFactory factory) {
+    public AttachmentListItemAdapter(Context context,
+                                     @NonNull MessageListItem.MessageItem messageListItem,
+                                     @NonNull MessageViewHolderFactory factory) {
         this.context = context;
         this.messageListItem = messageListItem;
         this.message = messageListItem.getMessage();
         this.factory = factory;
-        this.attachments = message.getAttachments();
+        this.attachments = CollectionsKt.map(message.getAttachments(), AttachmentListItem::new);
```

Comment:
Example of how handy Kotlin collection functions are, even when writing Java code

Reply:
I have never used it from Java, Taking note ;)